### PR TITLE
Ems 81643562 - Add group dashboard.

### PR DESCRIPTION
**Pull request by eschlange**
* Branch includes a new group dashboard page.
* Includes weekly aggregated breakdowns for participant created items.
* Includes weekly breakdown of logins by participant.
* Includes detailed tables for each item a participant created.

[Finished #81643562]

### CODE REVIEW

File: app/models/group.rb [hunk 1]
```diff
@@ -27,4 +27,129 @@ def learning_tasks
...
+    on_the_mind_statements_count
+  end
+
+  def likes_by_week(week_number)
+    likes_statements_count = 0
+    memberships.each do |membership|
+      likes_statements_count +=
+        SocialNetworking::Like
+        .where(participant: membership.participant)
+        .where("created_at >= ? AND created_at < ?",
```
Comment (michaelwehrley) [inline app/models/group.rb]: @eschlange You are going to hate me for this, but I went through a few weeks ago and changed all SQL calls to arel if possible.  Is it possible to refactor these to arel? :rabbit:

File: app/views/think_feel_do_engine/coach/group_dashboard/_comments.html.erb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            <th>Comment</th>
+            <th>Commented On</th>
+            <th>Created At</th>
+            <th>Week</th>
+          </tr>
+          </thead>
+          <tbody>
+
+          <% group.memberships.each do |membership| %>
+            <% SocialNetworking::Comment.where(participant: membership.participant).each do |comment| %>
```
Comment (michaelwehrley) [inline app/views/think_feel_do_engine/coach/group_dashboard/_comments.html.erb]: @eschlange do we want to be putting these query statements (as well as the goals below) in the view?  Is it possible to define a ```comments``` or ```goals``` method on a participant?  Maybe a participant Concern with those calls?

**Comment by michaelwehrley** [comment]
:+1:

File: app/models/group.rb (hunk 1 above)
Comment (eschlange) [inline app/models/group.rb]: All raw SQL in groups.rb and memberships.rb was converted to AREL syntax in the latest push.

File: app/views/think_feel_do_engine/coach/group_dashboard/_comments.html.erb (hunk 2 above)
Comment (eschlange) [inline app/views/think_feel_do_engine/coach/group_dashboard/_comments.html.erb]: Moved comments and goals to methods within the membership model to remove data level abstractions from the view.

**Comment by michaelwehrley** [comment]
:+ :camel: :cactus:

**Pull request merged**